### PR TITLE
Dockerstatic: Add sles12 and sles15 dockerfiles

### PR DESCRIPTION
**Pull request by Haroon-Khel**
- [x] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [x] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

~Draft~ pr for sles12 and sles15 dockerfiles to be hosted on dockerhost-marist-ubuntu2204-s390x-1

ref https://github.com/adoptium/infrastructure/issues/2689

### CODE REVIEW

**Comment by Haroon-Khel** [comment]
Turns out the sles12 box can pull from sles 15 repositories. @jiekang ref the possible kernel update; turns out docker containers use the host kernel anyway, so I do not think we have to worry about the sles15 repos updating the kernel here

**Comment by Haroon-Khel** [comment]
Got a sles12 box running https://ci.adoptopenjdk.net/computer/test-docker-sles12-s390x-1/ and a sles15 box running https://ci.adoptopenjdk.net/computer/test-docker-sles15-s390x-1/, but neither have labels yet. 

Sanity grinders running on both machines have shown only util tests to fail
https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/5688/ (sles 12)

**Comment by Haroon-Khel** [comment]
ping @sxa @steelhead31 for review

**Review by steelhead31 — APPROVED** [review]
LGTM

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.sles12 [hunk 1]
```diff
@@ -0,0 +1,38 @@
+FROM registry.suse.com/suse/sles12sp5:latest
+
+RUN zypper ar https://download.opensuse.org/distribution/leap/15.4/repo/oss/ sles15oss
+RUN zypper --gpg-auto-import-keys refresh
+RUN zypper update -y && zypper install -y wget perl openssh-server unzip zip tar gzip
+
+RUN wget 'https://api.adoptium.net/v3/binary/latest/11/ga/linux/s390x/jdk/hotspot/normal/eclipse?project=jdk' -O /tmp/jdk11.tar.gz
```
Comment (zdtsw) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.sles12:7]: any reason we use 11/ga/linux/s390x ?

Comment (Haroon-Khel) [reply ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.sles12:7]: Our other dockerstatic dockerfiles use jdk 8, but we dont release that for s390x, so I've used 11

**Review by zdtsw — APPROVED** [review]

**Pull request merged**